x-pack/plugins/elastic_assistant/server/__mocks__/alerts.ts

```diff
@@ -0,0 +1,3210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Mock alerts in the JSON format returned by the `fields` API, with the
+ * following distribution of severity:
+ *
+ * 1x severity: 'low',
+ * 1x severity: 'medium'
+ * 2x severity: 'high',
+ * 1x severity: 'critical',
+ */
+export const mockAlertsFieldsApi = [
```

Comment:
nite: lotta alert data, if uniqueness not super necessary maybe templatize one and modify severity with accessor?